Sources/Snowplow/Ecommerce/Entities/EcommScreenEntity.swift

```diff
@@ -0,0 +1,55 @@
+//  Copyright (c) 2013-2023 Snowplow Analytics Ltd. All rights reserved.
+//
+//  This program is licensed to you under the Apache License Version 2.0,
+//  and you may not use this file except in compliance with the Apache License
+//  Version 2.0. You may obtain a copy of the Apache License Version 2.0 at
+//  http://www.apache.org/licenses/LICENSE-2.0.
+//
+//  Unless required by applicable law or agreed to in writing,
+//  software distributed under the Apache License Version 2.0 is distributed on
+//  an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+//  express or implied. See the Apache License Version 2.0 for the specific
+//  language governing permissions and limitations there under.
+
+import Foundation
+
+/**
+ Attach Ecommerce Screen (Page) details to events.
+ Entity schema: `iglu:com.snowplowanalytics.snowplow.ecommerce/page/jsonschema/1-0-0`
+ */
+@objc(SPEcommScreenEntity)
+public class EcommScreenEntity: NSObject {
```

Comment:
Nit: I'm not a fan of abbreviations, I would prefer to call this `EcommerceScreenEntity` (and the same for the user one).